src/app/pages/listing/listing-edit-upload.jsx

```diff
@@ -457,24 +320,150 @@ function ChplListingEditUploadPage({ id }) {
                 </CardContent>
               </Card>
             </ListingContext.Provider>
-          </div>
-          <div>
-            { newListing
-              && (
-                <ListingContext.Provider value={newListingState}>
-                  <Card>
-                    <Box className={classes.sectionHeader}>
-                      <Typography className={classes.sectionHeaderText} variant="h2">Additional Information</Typography>
-                    </Box>
-                    <CardContent>
-                      <ChplAdditionalInformation
-                        listing={newListing}
-                      />
-                    </CardContent>
-                  </Card>
-                </ListingContext.Provider>
+          </Box>
+          </Box>
+          <Box>{/* Upload Container */}
+            <Typography gutterBottom component={"h3"} style={{ fontWeight: '600' }} variant='h4'>Updated Listing</Typography>
+            <div>
+              {!newListing ? (
+                <Box className={classes.placeholderContainer}>
+                  <HelpOutlineIcon fontSize="large" color="primary" />
+                  <Typography>Upload a file above to display your new listing.</Typography>
+                </Box>
+              ) : (
+                <>
+                  {newListing && (
+                    <Fade style={{ transitionDelay: newListing ? '1s' : '0ms' }} in={!!newListing}>
+                      <Box className={classes.listingContainer}>
+                        <ListingContext.Provider value={newListingState}>
+                          <Card>
+                            <Box className={classes.sectionHeader}>
+                              <Typography className={classes.sectionHeaderText} variant="h5">Listing Information</Typography>
+                            </Box>
+                            <CardContent>
+                              <ChplListingInformation listing={newListing} />
+                            </CardContent>
+                          </Card>
+                        </ListingContext.Provider>
+
+                        <ListingContext.Provider value={newListingState}>
```

Comment:
Do these components line up with the "original listing" elements? I had to build them in a staggered fashion to make sure each component lined up with the correlating element. Kind of like:

```
| original 1 | new 1 |
| original 2 | new 2 |
| original 3 | new 3 |
```
in order to get `original 3` to start at the same vertical place as `new 3`. In this, it looks like you have it as more like
```
| original 1 
original 2 
original 3 | new 1
new 2
new 3 |
```
which suggests that the components won't line up